pkg/utils/kubeclient/volume_claim.go

```diff
@@ -55,6 +55,7 @@ func GetMountInfoFromVolumeClaim(client client.Client, name, namespace string) (
 	if pv.Spec.CSI != nil && len(pv.Spec.CSI.VolumeAttributes) > 0 {
 		path = pv.Spec.CSI.VolumeAttributes[common.VolumeAttrFluidPath]
 		mountType = pv.Spec.CSI.VolumeAttributes[common.VolumeAttrMountType]
+		subpath = pv.Spec.CSI.VolumeAttributes[common.VolumeAttrFluidSubPath]
```

Comment:
I think the subpath doesn't exist by default. If common.VolumeAttrFluidSubPath is empty, then subpath shouldn't be set.

Reply:
if not exist, the subpath will be an empty string